.gitignore

```diff
@@ -54,3 +54,4 @@ coverage/
 .project
 .settings
 .classpath
+.vscode/
```

Comment:
ignore visual studio code config

Reply:
While I agree with adding `.vscode` to `.gitignore`, it probably should be done in it's own PR.

Should this PR be merged in, but reverted, it would also revert this change.

Reply:
undone